pkg/datapath/linux/probes/probes.go

```diff
@@ -155,7 +155,6 @@ type MapTypes struct {
 	HaveArrayOfMapsMapType         bool `json:"have_array_of_maps_map_type"`
 	HaveHashOfMapsMapType          bool `json:"have_hash_of_maps_map_type"`
 	HaveDevmapMapType              bool `json:"have_devmap_map_type"`
-	HaveSockmapMapType             bool `json:"have_sockmap_map_type"`
```

Comment:
I'd leave this entry in place. It matches `bpftool` output and already contains other map types that aren't used by Cilium.